src/shorah/amplicon.py

```diff
@@ -348,7 +349,9 @@ def main(args):
     logging.debug('b2w returned %d', ret_b2w)
 
     # run diri_sampler on the aligned reads
-    win_file = 'w-%s-%d-%d.reads.fas' % (ref_name, reg_start, reg_stop)
+    win_file = 'w-%s-%u-%u.reads.fas' % (ref_name, reg_start, reg_stop)
```

Comment:
I would be in favour of writing file names with a single dot (it was my mistake many years ago), like

```
win_file = 'w-%s-%u-%u-reads.fas' % (ref_name, reg_start, reg_stop)
```

Reply:
Basically, that would be replacing all the searches for '*.reads*'  with '*-reads*'.

But my main concern would be the change of names compared to legacy data.